### ISSUE TITLE
Why not aria-select=true on the selected dates?

### ISSUE THREAD

**Issue by cloud-walker** [rehookify/datepicker#20]
Hi! I'm trying out this library, and when it comes to styling, I expected it to use aria-* (or at least data-*) attributes to convey semantic meaning of each element, like for example a given cell may have `aria-selected=true` when selected and `data-first / data-last` when is the first and last date of the range.

I think this way the datepicker would be:

1. more accessible
2. more easily stylizable (for example with tailwind I can simply put a classname like "aria-selected:bg-gray-600")

**Comment by Feshchenko** [comment]
@cloud-walker, hi, thanks for the issue submission.
For now, you can extend it by getting `selected` and `range` from the day object and pass them to the prop-getter

```
<button {...dayButton(day, {  ...(day.selected ? { 'aria-selected': true } : {} )}>{day.day}</button>
```

**Comment by cloud-walker** [comment]
Yes, thanks @Feshchenko, but what do you think about the suggestion?

**Comment by Feshchenko** [comment]
My first intention was to add all `aria-*` attributes to the prop-getters but it was not on the priority list.
I'm not sure about additional logic over `range-start`/`range-end` attributes.
I created separate class-name-utils for handling classNames for the range case in the [Examples](https://codesandbox.io/p/sandbox/range-with-context-hooks-fyv72j?file=%2Fsrc%2Fclassnames-utils.ts%3A1%2C1)

**Comment by Feshchenko** [comment]
@cloud-walker, hi.
In [v5.1.0](https://github.com/rehookify/datepicker/tree/v5.1.0) all selected entities get `aria-selected` attribute fro prop-getter
